### PR TITLE
create geoblock.sh

### PR DESCRIPTION
**Pull request by szaimen**
fix https://github.com/nextcloud/vm/issues/1519

### CODE REVIEW

**Comment by szaimen** [comment]
@XSR1 can you please test this?

**Comment by XSR1** [comment]
I have tested this. By default GeoIPEnable Off (/etc/apache2/mods-available/geoip.conf).
So can we add after this line: https://github.com/nextcloud/vm/blob/1d8127bb693359c57c82263419260ec1d60b4a69/network/geoblock.sh#L148

```
<IfModule mod_geoip.c>
  GeoIPEnable On
  GeoIPDBFile /usr/share/GeoIP/GeoIP.dat
  GeoIPDBFile /usr/share/GeoIP/GeoIPv6.dat
</IfModule>
```

This will read both IPv4 and IPv6 DAT files.
I think IPv4 is working fine, but IPv6 needs "_V6" in the end:

`SetEnvIf GEOIP_CONTINENT_CODE_V6 EU AllowCountryOrContinent`
`SetEnvIf GEOIP_COUNTRY_CODE_V6 FI AllowCountryOrContinent`

Source: [https://dev.maxmind.com/geoip/legacy/mod_geoip2/](url)

Maybe in future we need to add updated DAT files (or single file).
I havent tested yet: https://www.miyuru.lk/geoiplegacy

Any way great job! This sript is working great!

**Comment by szaimen** [comment]
Hey XSR1, thanks for testing! I'll have a look how to implement your findings later! Have a great day!

**Comment by szaimen** [comment]
@XSR1 I've addressed the review except the database update. I hope this is okay for now.
I would propose to add the database update later on.

BTW: I've found this:
https://changelogs.ubuntu.com/changelogs/pool/universe/g/geoip-database/geoip-database_20191224-2/changelog
https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/
https://blog.maxmind.com/2020/06/01/retirement-of-geoip-legacy-downloadable-databases-in-may-2022/
So in conclusion we seem to migrate to a different mod then. But for now I think this is good to go!

**Comment by XSR1** [comment]
@szaimen I think this is perfect for now, and with updated DAT files there no no need to rush to geoip2. 
Bytheway I tested https://dl.miyuru.lk/geoip/maxmind/country/maxmind.dat.gz and its working fine. It contains both IPv4 and IPv6 files.

Thanks

**Comment by szaimen** [comment]
I will implement the updating of the .dat files with 
https://github.com/nextcloud/vm/issues/1552
So for know I think this PR is ready.

**Comment by enoch85** [comment]
since @XSR1 have tested and says it's fine, let's merge.
